packages/catalog-realm/TODO.md

```diff
@@ -0,0 +1 @@
+Make sure to lint for hbs when we add gts files to this realm like we do in the experiments realm. Currently linting for templates fails because there are no gts files in this realm.
```

Comment:
Can't tell if this is meant to be a real file or your notes to self

Reply:
it's a real TODO. as soon as we actually put gts files in in this realm (which we will), we need to turn on linting. we can remove this TODO after that.

Reply:
it would suck to forget to lint this realm...